internal/ingress/controller/parser/translate.go

```diff
@@ -373,3 +519,39 @@ func knativeSelectSplit(splits []knative.IngressBackendSplit) knative.IngressBac
 	}
 	return res
 }
+
+func pathsFromK8s(path string, pathType networkingv1.PathType) []*string {
+	switch pathType {
+	case networkingv1.PathTypePrefix:
+		base := strings.Trim(path, "/")
+		if base == "" {
+			return kong.StringSlice("/")
+		}
+		return kong.StringSlice(
+			"/"+base+"$",
+			"/"+base+"/",
+		)
+	case networkingv1.PathTypeExact:
+		relative := strings.TrimLeft(path, "/")
+		return kong.StringSlice("/" + relative + "$")
+	default: // includes ImplementationSpecific
```

Comment:
Can we instead add a case for ImplementationSpecific and then add an error in the default case?

Reply:
Sure. I'll do that in one go with the other fixes once I hear back from you.

Reply:
99a6ae9